source/federation.rst

```diff
@@ -0,0 +1,410 @@
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+| LocalUser                 | ``varchar(255)`` | No   | See Cloud Usage Record          |                                       |
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+| LocalGroup                | ``varchar(255)`` | No   | See Cloud Usage Record          |                                       |
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+| GlobalUserName            | ``varchar(255)`` | No   | See Cloud Usage Record          |                                       |
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+| FQAN                      | ``varchar(255)`` | No   | See Cloud Usage Record          |                                       |
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+| IPVersion                 | ``byte``         | No   | 4 or 6                          |                                       |
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+| IPCount                   | ``int(11)``      | No   | The number of IP addresses of   |                                       |
+|                           |                  |      | IPType this user currently      |                                       |
+|                           |                  |      | assigned to them                |                                       |
++---------------------------+------------------+------+---------------------------------+---------------------------------------+
+
+A JSON schema defining a valid Public IP Usage message can be found at: https://github.com/apel/apel/blob/9476bd86424f6162c3b87b6daf6b4270ceb8fea6/apel/db/__init__.py
+
+APEL and accounting portal 
+::::::::::::::::::::::::::
+
+Once generated, records are delivered to the central accounting repository
+using APEL SSM (Secure STOMP Messenger). SSM client packages can be obtained
+at https://apel.github.io. A Cloud Accounting Summary Usage Record has also
+been defined and summaries created on a daily basis from all the accounting
+records received from the Resource Providers are sent to the EGI Accounting
+Portal. The `Accounting portal <http://accounting.egi.eu/>`_ also runs SSM to
+receive these summaries and provides a web view of the accounting data received
+from the Resource Providers.
+
+Accounting Probes
+:::::::::::::::::
+
+Implementation of the extactor probes for accounting are listed below:
+
+* OpenNebula: https://github.com/EGI-FCTF/oneacct_export
```

Comment:
This repository too should be migrated to EGI-Foundation

Reply:
Linked the right project